packages/app/src/app/vscode/extensionHostWorker/services/searchService.ts

```diff
@@ -11,15 +11,6 @@ initializeGlobals();
 
 async function initialize() {
   await initializeBrowserFS({ syncSandbox: true, syncTypes: true });
-
-  self.addEventListener('message', e => {
```

Comment:
Why is this removed?

Reply:
The listener wasn't doing anything

```ts
self.addEventListener('message', e => {	
    if (e.data.$type === 'input-write') {	
      const { $type, $data } = e.data.$data;	
      if ($type === 'file-search') {	
        const { query } = $data;	
      }	
    }	
  });
```

Reply:
That's true, I'd prefer to keep it there since I was planning on using this for the search implementation.

Reply:
OK I'll revert it and just disable the following line then 👍
```ts
const { query } = $data;
```